inductiva/__init__.py

```diff
@@ -66,3 +66,33 @@ def _check_for_available_package_update():
 
 
 _check_for_available_package_update()
+
+
+def _supports_ansi():
+    """Checks if we support ansi formatting for colors and bolds"""
+    user_disable_ansi = utils.format_utils.getenv_bool("INDUCTIVA_DISABLE_ANSI",
+                                                       False)
+    if sys.platform.startswith("win"):
+        return "TERM" in os.environ and os.environ[
+            "TERM"] == "xterm" and not user_disable_ansi
+    return hasattr(sys.stdout,
+                   "isatty") and sys.stdout.isatty() and not user_disable_ansi
+
+
+def no_formatter(x, *_):
+    """Indentity formatter, i.e, applies no formatting"""
```

Comment:
typo *Identity

Reply:
These formatter providers should go to the `utils.format_utils` module.